css/recommendations.css

```diff
@@ -0,0 +1,109 @@
+/* #RecommendationTray {
+  display:grid;
+  grid-template-columns:repeat(auto-fill, minmax(250px, 1fr));
+} */
+
+#RecommendationTray {
+  background-color: white;
+  padding-left: 15px;
+  padding-right: 15px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#RecommendationTray > div{
+  padding:5px;
+  width: 300px;
+  margin-left: auto;
```

Comment:
Do you know that you can write `margin-left` `margin-right` `margin-top` and `margin-bottom` in one line using the shorthand property? https://www.w3schools.com/css/css_margin.asp
You could do that with every css attribute that has `-left`, `-right` etc. Please do that to have shorter CSS files.